<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralPage</class>
 <widget class="QWidget" name="GeneralPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>465</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="startupGBox">
     <property name="title">
      <string>Startup</string>
     </property>
     <layout class="QGridLayout" name="startupGBoxGLayout">
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkRunRetroshareAtSystemStartup">
        <property name="text">
         <string>Start RetroShare when my system starts</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="checkStartMinimized">
        <property name="text">
         <string>Start minimized</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="runRetroshareAtSystemStartupMinimizedHLayout">
        <item>
         <spacer name="runRetroshareAtSystemStartupMinimizedHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>15</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chkRunRetroshareAtSystemStartupMinimized">
          <property name="text">
           <string>Start minimized on system start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="advGBox">
     <property name="title">
      <string>For Advanced Users </string>
     </property>
     <layout class="QVBoxLayout" name="advGBoxVLayout">
      <item>
       <widget class="QCheckBox" name="checkAdvanced">
        <property name="text">
         <string>Enable Advanced Mode (Restart Required)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="miscGBox">
     <property name="title">
      <string>Misc</string>
     </property>
     <layout class="QGridLayout" name="miscGBoxGLayout">
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkQuit">
        <property name="text">
         <string>Do not show the Quit RetroShare MessageBox</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkCloseToTray">
        <property name="text">
         <string>Minimize to Tray Icon</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="autoLogin">
        <property name="text">
         <string>Auto Login</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="adminHLayout">
        <item>
         <widget class="QCheckBox" name="enableRetroShareProtocol">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>Register retroshare:// as URL protocol (Restart required)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="adminLabel">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="toolTip">
           <string>You need administrator rights to change this option.</string>
          </property>
          <property name="pixmap">
           <pixmap resource="../images.qrc">:/images/admin-16.png</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="idleGBox">
     <property name="title">
      <string>Idle</string>
     </property>
     <layout class="QHBoxLayout" name="idleGBoxHLayout">
      <item>
       <widget class="QLabel" name="idleLabel">
        <property name="text">
         <string>Idle Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="idleSpinBox">
        <property name="suffix">
         <string> seconds</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="mainVSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="runStartWizard_PB">
     <property name="text">
      <string>Launch startup wizard</string>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/images/tools_wizard.png</normaloff>:/images/tools_wizard.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
